packages/dashboard-core-plugins/src/panels/index.ts

```diff
@@ -17,7 +17,9 @@ export { default as MarkdownPanel } from './MarkdownPanel';
 export { default as NotebookPanel } from './NotebookPanel';
 export { default as PandasPanel } from './PandasPanel';
 export * from './PandasPanel';
-export { default as Panel } from './Panel';
+export { default as CorePanel } from './CorePanel';
+// Deprecated - use CorePanel instead
+export { default as Panel } from './CorePanel';
```

Comment:
Same comment about importing and renaming for the deprecated export w/ a deprecated tag.

`CorePanel` is what `Panel` used to be right? Why would someone need/want `Panel` from `@deephaven/dashboard`? dh.ui would still need the dashboard-core-components version, right? Or does nothing there need the session open/close events?

Reply:
deephaven.ui doesn't use the `onSessionClosed` or `onSessionOpened` props, it only needs the BasePanel